### PR TITLE
PEP 595: expand and rewrite section

### PR DESCRIPTION
**Pull request by ezio-melotti**
I noticed I had a couple of commits in a local branch that I hadn't pushed/merged, so here they are.

### CODE REVIEW

**Review by gvanrossum — COMMENTED** [review]
Actually I believe that in .rst files code fragments should use double backticks.

File: pep-0595.rst [hunk 1]
```diff
@@ -370,7 +370,7 @@ need to be addressed regardless of the approach used:
 
   Since Roundup converts references to links when messages are
   requested, it is possible to update the target and generate the
-  correct link.  This need already arose several times, for
+  correct link.  This need already arised several times, for
```
Comment (gvanrossum) [inline pep-0595.rst]: Actually, "arose" is correct here. So it seems there's nothing to do in this PR?

**Comment by ezio-melotti** [comment]
The PR includes two commits with changes that for some reason don't show up in the diff unless the two commits are selected individually (the changes seem to be there though).

When I merged with master I got the following conflicts, and I think I accidentally kept the version with the single backticks (fixed now).  `arose` didn't show up in the diff, so I didn't notice it until it showed up here (also fixed now).

This is the output I got during the merge:
```
$ git merge master
Removing pep2pyramid.py
Auto-merging pep-0595.rst
CONFLICT (content): Merge conflict in pep-0595.rst
Removing pep-0458-3.png
Removing pep-0458-2.png
Removing .travis.yml
Automatic merge failed; fix conflicts and then commit the result.

$ git diff
diff --cc pep-0595.rst
index 98b2c82e,61964da0..00000000
--- a/pep-0595.rst
+++ b/pep-0595.rst
@@@ -291,7 -300,7 +300,11 @@@ only when someone wants to keep workin
  has several issues, but there are also other issues that will
  need to be addressed regardless of the approach used:
  
++<<<<<<< HEAD
 +* **Vendor lock-in.**  GitHub is properietary and there is risk
++=======
+ * **Vendor lock-in.**  GitHub is proprietary and there is risk
++>>>>>>> master
    of vendor lock-in.  Their business model might change and they
    could shut down altogether.  For example, several projects
    decided to move away from GitHub after Microsoft acquisition.
@@@ -337,11 -346,11 +350,19 @@@
    Setting up a redirect for each migrated issue on bpo might
    mitigate the issue, however -- if references in migrated messages
    are not updated -- it will cause confusion (e.g. if bpo issue
++<<<<<<< HEAD
 +  `#1234` becomes GitHub issue `#4321`, a reference to `#1234`
 +  in a migrated message could link to bpo `#1234` and bpo can
 +  redirect to GitHub issue `#4321`, but new references to `#1234`
 +  will link to GitHub PR `#1234` rather than GitHub issue `#4321`).
 +  Manually having to specify a `bpo-` or `gh-` prefix is error prone.
++=======
+   ``#1234`` becomes GitHub issue ``#4321``, a reference to ``#1234``
+   in a migrated message could link to bpo ``#1234`` and bpo can
+   redirect to GitHub issue ``#4321``, but new references to ``#1234``
+   will link to GitHub PR ``#1234`` rather than GitHub issue ``#4321``).
+   Manually having to specify a ``bpo-`` or ``gh-`` prefix is error prone.
++>>>>>>> master

[manually resolved the conflicts]

$ git diff
diff --cc pep-0595.rst
index 98b2c82e,61964da0..00000000
--- a/pep-0595.rst
+++ b/pep-0595.rst

$
```

**Comment by gvanrossum** [comment]
Maybe start over with a new PR and copying the file to a temp dir?

**Comment by ezio-melotti** [comment]
I double checked and it looks like all the changes are already present in the PEP, so this can be closed.  Sorry for the noise :)

**Comment by ezio-melotti** [comment]
FTR, the changes that I was trying to push were merged in b3f4c8e.

**Pull request closed without merging**